configuration/src/configuration.ts

```diff
@@ -113,6 +122,15 @@ export class Configuration implements IConfiguration {
         if (params.defaultValues !== null) {
             this.providers.push(defaultConfig);
         }
+
+        // Optionally add the keyvault provider:

```

Comment:
keyvault being one of the providers means we can put all our configuration into keyvault?